LEGO1/legoinputmanager.h

```diff
@@ -28,10 +28,26 @@ class LegoInputManager : public MxPresenter
 
   virtual MxResult Tickle() override; // vtable+0x8
 
-  undefined m_pad40[0x15c];
+  undefined m_pad40[0x48];
+
+  MxBool m_unk88;
+  MxU8 m_unk89;
+  MxU8 m_unk8a;
+  MxU8 m_unk8b;
+
+  undefined m_pad8c[0x110];
+
+  // 0x19C
   int m_joystickIndex;
-  undefined m_pad200[0x194];
+
+  undefined m_pad1a0[0x194];
+
+  // 0x334
   MxBool m_useJoystick;
+
+  MxU8 m_unk335;
```

Comment:
We should use `undefined` unless you have a good reason to believe those are `U8` (you might have, in which case it's fine, but there's no code checked in ATM that uses those)